transform_aes.c

```diff
@@ -208,12 +192,14 @@ static int transop_decode_aes( n2n_trans_op_t * arg,
             decode_uint32( &sa_rx, inbuf, &rem, &idx );
 
             /* Get the IV seed */
-            decode_buf((uint8_t *)&iv_seed, sizeof(iv_seed), inbuf, &rem, &idx);
+	    decode_buf(iv_seed, TRANSOP_AES_IV_SEED_SIZE, inbuf, &rem, &idx);
 
-            traceEvent( TRACE_DEBUG, "decode_aes %lu with seed %016lx", in_len, iv_seed );
+	    char * iv_seed_output[TRANSOP_AES_IV_SEED_SIZE*2];
+	    for (int i=0; i < TRANSOP_AES_IV_SEED_SIZE; i++) sprintf(iv_seed_output, "%s%02x", iv_seed_output,iv_seed[i]);
```

Comment:
Too many calls to sprintf here. Anyway, please remove and keep the old iv seed code for now

Reply:
Will do... I just hope I can manage git to accept my local changes to this pull request. I guess my remote HEAD changed meanwhile.... Let's see.